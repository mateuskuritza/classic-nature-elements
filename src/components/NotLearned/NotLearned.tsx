import styled from "styled-components";

export default function NotLearned() {
	return (
		<Container>
			<p>NÃ£o dominado</p>
		</Container>
	);
}

const Container = styled.div`
	background-color: red;
	color: white;
	width: 90%;
	text-align: center;
	padding: 2px;
	border-radius: 4px;
`;
