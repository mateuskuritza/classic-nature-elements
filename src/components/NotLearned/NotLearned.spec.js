import React from "react";
import { render, fireEvent, screen } from "@testing-library/react";
import NotLearned from "./NotLearned";

it("Should render NotLearned", () => {
	const { container } = render(<NotLearned />);
	const button = screen.getByText("NÃ£o dominado");
	expect(button).toBeDefined();
	expect(container).toMatchSnapshot();
});
