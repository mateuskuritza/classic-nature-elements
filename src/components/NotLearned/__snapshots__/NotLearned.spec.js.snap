// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render NotLearned 1`] = `
<div>
  <div
    class="sc-bdvvtL iEMdHO"
  >
    <p>
      NÃ£o dominado
    </p>
  </div>
</div>
`;
